### PR TITLE
tolerate any taint

### PR DESCRIPTION
**Pull request by deads2k**
/assign @abhinavdahiya @mfojtik

### CODE REVIEW

File: bindata/v3.11.0/kube-apiserver/deployment.yaml [hunk 1]
```diff
@@ -84,3 +84,5 @@ spec:
           secretName: serving-cert
```
Comment (abhinavdahiya) [inline bindata/v3.11.0/kube-apiserver/deployment.yaml]: Why is this file called `kube-apiserver/deployment.yaml` when its controller-manager ?

**Comment by abhinavdahiya** [comment]
`/lgtm`

The tolerate all is from https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/#concepts

File: bindata/v3.11.0/kube-apiserver/deployment.yaml (hunk 1 above)
Comment (deads2k) [reply bindata/v3.11.0/kube-apiserver/deployment.yaml]: > Why is this file called `kube-apiserver/deployment.yaml` when its controller-manager ?

It's content is a deployment

Comment (abhinavdahiya) [reply bindata/v3.11.0/kube-apiserver/deployment.yaml]: `deployment.yaml` seems fine, just that the directory was `kube-apiserver`?
Its an trivial curiosity.

**Pull request merged**